cloudbuild-tagged.yaml

```diff
@@ -0,0 +1,14 @@
+steps:
+- name: 'java:8'
```

Comment:
I bet there's a way to wire the yarn cache into a starting image... Something to look at for another day.